emission/tests/netTests/TestHabiticaRegister.py

```diff
@@ -58,6 +58,25 @@ def testGetUserProfile(self):
                         'healer': False})
       self.assertEqual(ret_json['data']['newMessages'], {})
 
+  def testJoinParty(self):
+      sampleAuthMessage1Ad = ad.AttrDict(self.sampleAuthMessage1)
+      proxy.habiticaRegister(sampleAuthMessage1Ad.username, sampleAuthMessage1Ad.email,
+                             sampleAuthMessage1Ad.password, sampleAuthMessage1Ad.our_uuid)
+      #Create an inviter
+      inviterUUID = uuid.uuid4()
+      inviter = randomGen()
+      inviter_email = inviter + '@save.world'
+      inviter_id = proxy.habiticaRegister(inviter, inviter_email, inviter, inviterUUID)['data']['id']
+      inviter_group_id = json.loads(proxy.habiticaProxy(inviterUUID, 'POST', "/api/v3/groups", {'type': 'party', 'privacy': 'private', 'name': inviter}).text)['data']['id']
+      #Finally try to make this user (self) join the party
+      group_id_returned = proxy.setup_party(self.testUserUUID, inviter_group_id, inviter_id)
+      self.assertEqual(group_id_returned, inviter_group_id)
+      #Now try to join again, it should throw an error
+      #proxy.setup_party(self.testUserUUID, inviter_group_id, inviter_id)
```

Comment:
The classic way to deal with that is to use an `assertRaises` .
https://github.com/e-mission/e-mission-server/blob/2a6087035b3a897879067c36d24f0ba97a6e0c79/emission/tests/storageTests/TestTimeSeries.py#L68